File: ruby/fizzbuzz.rb
class FizzBuzz 

	def initialize(size)
		@x = size
	end

	def fizzbuzz()
		(1..@x).map { |n| 
			a = "#{'Fizz' if n%3==0}#{'Buzz' if n%5==0}"
			a=='' ? "#{n}" : a
		}
	end	
end

File: src/main/kotlin/FizzBuzz.kt

class FizzBuzz (val size : Int){
    fun fizzBuzz(): List<String> {
        return (1..size).map {
            when {
                it % 15 == 0 -> "FizzBuzz"
                it % 3 == 0 -> "Fizz"
                it % 5 == 0 -> "Buzz"
                else -> it.toString()
            }
        }
    }
}